scripts/single-node.sh

```diff
@@ -41,6 +41,9 @@ createGenesis() {
       --home "${APP_HOME}" \
       > /dev/null 2>&1 # Hide output to reduce terminal noise
 
+    echo "Overriding the genesis.json expedited min deposit from 50 TIA to 50,000 TIA."
+    sed -i'.bak' 's#"50000000"#"50000000000"#g' "${APP_HOME}"/config/genesis.json
```

Comment:
This is based on https://github.com/celestiaorg/celestia-app/issues/4742#issuecomment-2848067911